### PR TITLE
fix: small screen overflowing issue

### PR DESCRIPTION
**Pull request by B-Mustafa**

### CODE REVIEW

**Comment by B-Mustafa** [comment]
https://github.com/brianlovin/briOS/issues/2024

fixed this issue

**Review by brianlovin — COMMENTED** [review]
Thanks for taking a look at this! Is there a reason for adding extra spaces around some of the class names?

**Comment by B-Mustafa** [comment]
No , not really , i was checking out and might have left out left whitespaces

**Comment by B-Mustafa** [comment]
Any updates on it @brianlovin

**Comment by B-Mustafa** [comment]
hey @brianlovin , any updates regarding this PR ?

**Comment by brianlovin** [comment]
@B-Mustafa — could you push a change to clear out the unneeded space characters in the diff? Thank you!

**Comment by B-Mustafa** [comment]
Alright i will push a new changes one without extra spaces !

**Comment by B-Mustafa** [comment]
@brianlovin did the latest commit with all the necessary changes
small screen

**Comment by B-Mustafa** [comment]
@brianlovin any updates ?

File: src/components/Home/Intro.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ function SectionTitle(props) {
 }
 
 function SectionContent(props) {
-  return <div className="col-span-10" {...props} />
+  return <div className="col-span-10 " {...props} />
```
Comment (brianlovin) [inline src/components/Home/Intro.tsx:20]:
```suggestion
  return <div className="col-span-10" {...props} />
```

File: src/components/Home/Intro.tsx [hunk 2]
```diff
@@ -351,7 +351,7 @@ export function Intro() {
           <SectionContainer>
             <SectionTitle>Work</SectionTitle>
             <SectionContent>
-              <div className="flex flex-col space-y-3">
+              <div className="flex flex-col space-y-3 ">
```
Comment (brianlovin) [inline src/components/Home/Intro.tsx:354]:
```suggestion
              <div className="flex flex-col space-y-3">
```

File: src/components/Home/Intro.tsx [hunk 3]
```diff
@@ -368,7 +368,7 @@ export function Intro() {
           <SectionContainer>
             <SectionTitle>Speaking</SectionTitle>
             <SectionContent>
-              <div className="flex flex-col space-y-3">
+              <div className="flex flex-col space-y-3 ">
```
Comment (brianlovin) [inline src/components/Home/Intro.tsx:371]:
```suggestion
              <div className="flex flex-col space-y-3">
```

**Review by brianlovin — APPROVED** [review]

**Comment by brianlovin** [comment]
Merged @B-Mustafa — thank you!

**Comment by brianlovin** [comment]
Needed to revert, but fixed the responsiveness and typography in later commits! Thanks again for the nudge.

**Comment by B-Mustafa** [comment]
@brianlovin alright brian Thanks for the Update

**Pull request merged**